README.md

```diff
@@ -1,2 +1,63 @@
-# open-pandemic
-Open Pandemic collects information from different data sources to provide comprehensive data for the novel coronavirus, SARS-CoV-2.
+# Open Pandemic
+
+> Open Pandemic is a single page web application that consumes the Narrative API to get real time Covid 19 data. It lists recorded Covid 19 cases by country.
+
+> ![screenshot](./src/images/app1.png) ![screenshot](./src/images/app2.png)
+
+Users can interact with the app by filtering the list of countries by name.
+
+## Languages
+
+- HTML/CSS
+- JavaScript
+
+## Technologies
+
+- React Bootstrap
+- React
+- Redux
+
+## Live Demo
+
+[Live Project](https://open-pandemic.herokuapp.com/)
```

Comment:
- [ ] The deployed version isn't working properly, kindly check these console errors.
![image](https://user-images.githubusercontent.com/33767951/154595579-62224928-2460-4f3f-88b8-f8a707aeeb5b.png)